scripts/update-nightly/updateSubmodules.mjs

```diff
@@ -78,6 +79,9 @@ async function updateDocs(versions, isNightly) {
         case 'wallet':
           submoduleName = `docs${isNightly ? '/nightly' : ''}/fuels-wallet`;
           break;
+        case 'sway_standards':
```

Comment:
Do we need a check for sway libs as well?

Reply:
yes good catch ty